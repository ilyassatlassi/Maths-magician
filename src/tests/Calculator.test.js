import React from "react";
import MathsMagician from "../component/pages/Calculator";
import renderer from "react-test-renderer";

describe("Calculator Component", () => {
  it("should render correctly", () => {
    const tree = renderer.create(<MathsMagician />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
