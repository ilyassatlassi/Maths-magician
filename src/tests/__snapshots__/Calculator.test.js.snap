// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Calculator Component should render correctly 1`] = `
<div
  className="App"
>
  <div
    className="cacul-section"
  >
    <div
      className="input-numbers"
    >
      <input
        readOnly={true}
        type="text"
        value="0"
      />
    </div>
    <div
      className="content"
    >
      <div
        className="function-calcul"
      >
        <div
          className="top-calcul"
        >
          <button
            onClick={[Function]}
            type="button"
          >
            AC
          </button>
          <button
            onClick={[Function]}
            type="button"
          >
            +/-
          </button>
          <button
            onClick={[Function]}
            type="button"
          >
            %
          </button>
        </div>
        <div
          className="numbers"
        >
          <button
            onClick={[Function]}
            type="button"
          >
            7
          </button>
          <button
            onClick={[Function]}
            type="button"
          >
            8
          </button>
          <button
            onClick={[Function]}
            type="button"
          >
            9
          </button>
          <button
            onClick={[Function]}
            type="button"
          >
            4
          </button>
          <button
            onClick={[Function]}
            type="button"
          >
            5
          </button>
          <button
            onClick={[Function]}
            type="button"
          >
            6
          </button>
          <button
            onClick={[Function]}
            type="button"
          >
            1
          </button>
          <button
            onClick={[Function]}
            type="button"
          >
            2
          </button>
          <button
            onClick={[Function]}
            type="button"
          >
            3
          </button>
        </div>
        <div
          className="bottom-calcul"
        >
          <button
            className="button0"
            onClick={[Function]}
            type="button"
          >
            0
          </button>
          <button
            onClick={[Function]}
            type="button"
          >
            .
          </button>
        </div>
      </div>
      <div
        className="operation"
      >
        <button
          onClick={[Function]}
          type="button"
        >
          รท
        </button>
        <button
          onClick={[Function]}
          type="button"
        >
          x
        </button>
        <button
          onClick={[Function]}
          type="button"
        >
          -
        </button>
        <button
          onClick={[Function]}
          type="button"
        >
          +
        </button>
        <button
          onClick={[Function]}
          type="button"
        >
          =
        </button>
      </div>
    </div>
  </div>
</div>
`;
